nimdoc/testproject/subdir/subdir_b/utils.nim

```diff
@@ -115,3 +115,8 @@ proc f*(x: G[int]) =
 proc f*(x: G[string]) =
   ## See also `f(G[int])`_.
   discard
+
+## Ref. `[]`_ is the same as `proc \`[]\`(G[T])`_ because there are no
+## overloads.
+
```

Comment:
can you add more tests especially for edge cases, eg:
```
## See `$`_, `[]=`_
## See `'big`_ (for user defined literals)
```

Reply:
added. Thanks for testcases, they helped to catch some bugs.